mParticle-Apple-SDK/mParticle.m

```diff
@@ -159,6 +159,89 @@ - (NSString *)description {
+        [[MParticle sharedInstance].kitContainer_PRIVATE forwardSDKCall:roktSelector
+                                                                  event:nil
+                                                             parameters:queueParameters
+                                                            messageType:MPMessageTypeEvent
+                                                               userInfo:nil
+        ];
+    });
+}
+
+- (void)selectPlacements:(NSString * _Nullable)identifier
+              attributes:(NSDictionary<NSString *, NSString *> * _Nullable)attributes
+              placements:(NSDictionary<NSString *, id> * _Nullable)placements
+                  onLoad:(void (^ _Nullable)(void))onLoad
+                onUnLoad:(void (^ _Nullable)(void))onUnLoad
+onShouldShowLoadingIndicator:(void (^ _Nullable)(void))onShouldShowLoadingIndicator
+onShouldHideLoadingIndicator:(void (^ _Nullable)(void))onShouldHideLoadingIndicator
+    onEmbeddedSizeChange:(void (^ _Nullable)(NSString * _Nonnull, CGFloat))onEmbeddedSizeChange {
+    NSDictionary<NSString *, NSString *> *mappedAttributes = [MPRokt mapUserAttributes:attributes mappingDictionary:@{}];
+    for (NSString *key in mappedAttributes) {
+        [[MParticle sharedInstance].identity.currentUser setUserAttribute:key value:mappedAttributes[key]];
+    }
+    
+    dispatch_async(dispatch_get_main_queue(), ^{
+        // Forwarding call to kits
+        MPForwardQueueParameters *queueParameters = [[MPForwardQueueParameters alloc] init];
+        [queueParameters addParameter:identifier];
+        [queueParameters addParameter:attributes];
+        [queueParameters addParameter:placements];
+        [queueParameters addParameter:onLoad];
+        [queueParameters addParameter:onUnLoad];
+        [queueParameters addParameter:onShouldShowLoadingIndicator];
+        [queueParameters addParameter:onShouldHideLoadingIndicator];
+        [queueParameters addParameter:onEmbeddedSizeChange];
+        
+        SEL roktSelector = @selector(executeWithViewName:attributes:placements:onLoad:onUnLoad:onShouldShowLoadingIndicator:onShouldHideLoadingIndicator:onEmbeddedSizeChange:filteredUser:);
+        [[MParticle sharedInstance].kitContainer_PRIVATE forwardSDKCall:roktSelector
+                                                                  event:nil
+                                                             parameters:queueParameters
+                                                            messageType:MPMessageTypeEvent
+                                                               userInfo:nil
+        ];
+    });
+}
+
++ (NSDictionary<NSString *, NSString *> *) mapUserAttributes:(NSDictionary<NSString *, NSString *> * _Nonnull)attributes mappingDictionary:(NSDictionary<NSString *, NSString *> *)mappingDictionary {
```

Comment:
colon align

Reply:
rarely do we colon align with only 2 parameters but I do agree it looks nice so 🤷